mmv1/api/resource.rb

```diff
@@ -201,6 +201,12 @@ module Properties
       # instead of a project field to use User Project Overrides
       attr_reader :supports_indirect_user_project_override
 
+      # If true, the resource's project field can be specified as either the short form project
+      # id or the long form projects/project-id. The extra projects/ string will be removed from
+      # urls and ids. This should only be used for resources that previously supported long form
+      # project ids for backwards compatibility.
+      attr_reader :long_form_project
```

Comment:
nit: Consider prefixing with `legacy_` to make it even more clear we shouldn't use this on newer resources.

Reply:
Done.